src/main/java/pro/beam/api/services/impl/ChannelsService.java

```diff
@@ -40,8 +40,8 @@ public ChannelsService(BeamAPI beam) {
             options.put("only", Enums.serializedName(only));
         }
 
-        options.put("page", Math.min(0, page));
-        options.put("limit", Math.min(0, limit));
+        options.put("page", Math.max(0, page));
+        options.put("limit", Math.max(0, limit));
```

Comment:
By using Math.min, the `page` and `limit` can only be <= 0 which is not what I want when im trying to scroll endlessly by paging :dancers: